ExtLibs/Maps/WPOverlay.cs

```diff
@@ -88,31 +92,28 @@ public void CreateOverlay(PointLatLngAlt home, List<Locationwp> missionitems, do
                         pointlist.Add(new PointLatLngAlt(item.lat, item.lng,
                             item.alt + gethomealt((MAVLink.MAV_FRAME) item.frame, item.lat, item.lng),
                             "DLS" + (a + 1).ToString()));
-                        route.Add(pointlist[pointlist.Count - 1]);
+                        AddPointToRoute(a + 1, pointlist[pointlist.Count - 1]);
 
                         dolandstart = a;
                         // draw everything before
                         if (route.Count > 0)
                         {
-                            RegenerateWPRoute(route, home, false);
-                            route.Clear();
+                            pointlist[pointlist.Count - 1].Tag = "break";

```

Comment:
This overwrites the "DLS" tag I'm adding here, which breaks the elevation profile for landing sequences with hold times. I know we don't use it right now, but let's not regress it. Should be able to just delete this bit and add a `Contains("DLS")` check later.